Sources/CwlUtils/CwlScalarScanner.swift

```diff
@@ -114,7 +114,7 @@ public struct ScalarScanner<C: Collection> where C.Iterator.Element == UnicodeSc
 	}
 	
 	/// Throw if the next scalar at the current `index` fails to match the next scalar in `scalars`. Advance the `index` to the end of the match.
-	public mutating func match(where test: (UnicodeScalar) -> Bool) throws {
+	public mutating func match(where test: @escaping (UnicodeScalar) -> Bool) throws {
```

Comment:
We are marking `test` `@escaping` here but actually it doesn't. Maybe `String.init(describing:)` is implemented with `@autoclosure`, but I couldn't find the implementation details on header files.